remove direct placement (#7400)
pingcap/docs: placement-rules-in-sql.md

- Schedule the leaders of hotspot data to high-performance TiKV instances
- Separate cold data to lower-cost storage mediums to improve cost efficiency

## Specify placement rules

> The first step in specifying placement rules is to create a `PLACEMENT POLICY`:

Reviewer: ```suggestion
To specify placement rules, first create a placement policy:
```

```sql
CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1";
```

> This policy can then be attached to a table or partition using either `CREATE TABLE` or `ALTER TABLE`:

Reviewer: ```suggestion
Then attach the policy to a table or partition using either `CREATE TABLE` or `ALTER TABLE`:
```

```sql
CREATE TABLE t1 (a INT) PLACEMENT POLICY myplacementpolicy;
CREATE TABLE t2 (a INT);
ALTER TABLE t2 PLACEMENT POLICY myplacementpolicy;
```

> A `PLACEMENT POLICY` is not associated with any database schema and has the global scope. Therefore, assigning a placement policy does not require any additional privileges over the `CREATE TABLE` privilege.

Reviewer: ```suggestion
A placement policy is not associated with any database schema and has the global scope. Therefore, assigning a placement policy does not require any additional privileges over the `CREATE TABLE` privilege.
```

> ## Observing current placement rules

Reviewer: ```suggestion
## View current placement rules
```

> If a table has placement rules attached, they can be observed in the output of `SHOW CREATE TABLE`, with the definition of the policy available in `SHOW CREATE PLACEMENT POLICY`:

Reviewer: ```suggestion
If a table has placement rules attached, you can view the placement rules in the output of `SHOW CREATE TABLE`. To view the definition of the policy available, execute `SHOW CREATE PLACEMENT POLICY`:
```

```sql
tidb> SHOW CREATE TABLE t1\G
*************************** 1. row ***************************
       Table: t1
Create Table: CREATE TABLE `t1` (
  `a` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin /*T![placement] PLACEMENT POLICY=`myplacementpolicy` */
1 row in set (0.00 sec)

tidb> SHOW CREATE PLACEMENT POLICY myplacementpolicy\G
*************************** 1. row ***************************
       Policy: myplacementpolicy
Create Policy: CREATE PLACEMENT POLICY myplacementpolicy PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-west-1"
1 row in set (0.00 sec)
```

> The `information_schema` tables `tables` and `partitions` also include a column for `tidb_placement_policy_name`, which makes it possible to find all objects which have placement rules attached:

Reviewer: ```suggestion
The `information_schema.tables` and `information_schema.partitions` tables also include a column for `tidb_placement_policy_name`, which shows all objects with placement rules attached:
```
